### PR TITLE
Updating plugins to v1.2

### PR DESCRIPTION
**Pull request by psschwei**
Updating plugins as part of release for v1.2

/assign @rhuss @dsimansk

### CODE REVIEW

**Comment by cardil** [comment]
I think the `event@1.1.rb` with old version is missing.

File: admin.rb [hunk 1]
```diff
@@ -13,10 +13,10 @@ class Admin < Formula
 
   if OS.mac?
     url "#{base_url}/#{file_name}-darwin-amd64"
-    sha256 "a138a253b511aa9be49dc7b4e4380ef8351a4e96f66b86c32b62ad7af8728050"
+    sha256 "f4f94cda09c8fcd21b0a7c8adf0d57ee0e56c7ac026e2fd07d0954ff9abdbee7"
```
Comment (csantanapr) [inline admin.rb:16]: I would like to see this being calculated by automation script

Comment (psschwei) [reply admin.rb:16]: @cardil has done that for the event plugin, which I think is going to become the standard. [slack discussion](https://knative.slack.com/archives/C02SFDXD72A/p1643207831189200)

**Pull request merged**